Framework/MDAlgorithms/src/IntegratePeaksMDHKL.cpp

```diff
@@ -69,6 +69,20 @@ void IntegratePeaksMDHKL::init() {
                                                      Direction::Output),
       "The output PeaksWorkspace will be a copy of the input PeaksWorkspace "
       "with the peaks' integrated intensities.");
+  declareProperty(
+      make_unique<PropertyWithValue<double>>("BackgroundInnerRadius", 0.0,
```

Comment:
Using `EMPTY_DBL()` as default value will then show correctly that it's optional but you will need to change how it's handled later.